pkgs/development/compilers/dtc/default.nix

```diff
@@ -53,6 +53,12 @@ stdenv.mkDerivation (finalAttrs: {
       url = "https://git.kernel.org/pub/scm/utils/dtc/dtc.git/patch/?id=3fbfdd08afd2a7a25b27433f6f5678c0fe694721";
       hash = "sha256-skK8m1s4xkK6x9AqzxiEK+1uMEmS27dBI1CdEXNFTfU=";
     })
+
+    # pylibfdt: fix get_mem_rsv for newer Python versions
+    (fetchpatch {
+      url = "https://github.com/dgibson/dtc/commit/822123856980f84562406cc7bd1d4d6c2b8bc184.patch";
```

Comment:
I think the canonical repo is the one on kernel.org. (That's the one mentioned in the README.)

Reply:
I'm just trying to be consistent with the rest of the file here.

Reply:
It's already inconsistent — the patch directly above uses the kernel.org repo.

Reply:
(But it doesn't really matter, if the rest of the file is already using that repo…)